storage/tianmu/core/mysql_expression.cpp

```diff
@@ -608,6 +608,9 @@ bool operator==(Item const &l_, Item const &r_) {
         same = !std::strcmp(l->func_name(), r->func_name());
```

Comment:
refine the input params using google coding style. the input params or temp vars should not be with `_`.

Reply:
fixed it